ssl/tls1.h

```diff
@@ -528,6 +528,14 @@ SSL_CTX_callback_ctrl(ssl,SSL_CTRL_SET_TLSEXT_TICKET_KEY_CB,(void (*)(void))cb)
 #define TLS1_CK_ECDH_RSA_WITH_AES_128_GCM_SHA256        0x0300C031
 #define TLS1_CK_ECDH_RSA_WITH_AES_256_GCM_SHA384        0x0300C032
 
+/* Ring-LWE based ciphersuites */
+#define TLS1_CK_RLWE_RSA_WITH_AES_128_GCM_SHA256        0x0300D000
```

Comment:
It appears that you are trying to get an implementation into OpenSSL without properly specifying the protocol and getting number assignments from IANA; without any justification for such behavior, I would expect this pull request to be summarily closed.